SECURITY.md

```diff
@@ -0,0 +1,40 @@
+# Security policy
+
+## Security bulletins
+
+For information regarding the security of this project please join:
+
+* [konveyor slack channel](https://kubernetes.slack.com/archives/CR85S82A2)
+* [Mailing list](https://groups.google.com/u/0/g/konveyor-community)
+
+
+## Reporting a vulnerability
+
+Please use the below process to report a vulnerability to the project:
+
+1. Email the konveyor-security@googlegroups.com
+    * Emails should contain:
+        * description of the problem
+        * precise and detailed steps (include screenshots) that created the
+          problem
+        * the affected version(s)
+        * any possible mitigations, if known
+2. You will receive a reply from one of the maintainers within 2-4 days
+   acknowledging receipt of the email.
+3. You may be contacted by a member of Konveyor Maintainers to further discuss the reported item.
+   Please bear with us as we seek to understand the breadth and scope of the
+   reported problem, recreate it, and confirm if there is a vulnerability
+   present.
+4. As the issue passes through various stages like triage, code fix, 
+   release planning, the reporter will be kept informed. The reporter will be given credits during the disclosure.
+
+
+## Supported Versions
```

Comment:
Is 'supported versions' appropriate for us?

Is this where we would communicate longer lived versions that folks may want to file issues on and see resolved?

Reply:
I am hoping we have the supported versions in main tackle project and hyperlink it. That way its just one place to update and maintain. I left it as TODO for now but open to changes. WDYT?

Reply:
removed the table. Now, we just just hyperlink from the main repo. We definitely need to maintain a matrix of supported versions somewhere.

Reply:
You addressed my concern, I was unsure of how we would keep this list current.